build.gradle

```diff
@@ -27,14 +27,14 @@ project.ext {
     oltu_version = "1.0.1"
 }
 
-version = '2.1.0'
+version = '2.2.0'
```

Comment:
Not sure if it doesn't need to be 3.0.0 as it's not backwards compatible with the previous version...